superset/assets/javascripts/explorev2/stores/fields.js

```diff
@@ -1006,15 +1013,16 @@ export const fields = {
     type: 'TextField',
     label: 'Opacity',
     default: 1,
+    validators: [v.numeric],
```

Comment:
would be nice to add a more specific validator here to enforce integers between 0 and 1 only, but that can be added later now that you've got this framework in place.